rust-toolchain.toml

```
@@ -0,0 +1,2 @@
+[toolchain]
+channel = "stable"
```

```suggestion
channel = "1.80.0"
components = ["rust-analyzer"]
profile = "default"
```
stableと書いても固定にならない件について推していきたいです

---

確かに！
suggestion そのまま採用しました